src/module/apps/effects-panel.ts

```diff
@@ -99,21 +100,25 @@ export class EffectsPanel extends Application {
 
             const iconElem = effectEl.querySelector(":scope > .icon");
             // Increase or render persistent-damage dialog on left click
-            iconElem?.addEventListener("click", async () => {
+            iconElem?.addEventListener("click", async (event?: JQuery.TriggeredEvent | Event | null) => {
```

Comment:
```suggestion
            iconElem?.addEventListener("click", async (event) => {
```

Same for the other one.

Reply:
Thank you, if you have any idea how to better change crit increasing logic it would be great